node/packages/client/src/types.ts

```diff
@@ -54,6 +54,7 @@ export enum TxParser {
   UniV2 = 'uniV2',
   ZRX = 'zrx',
   Thor = 'thor',
+  Foxy = 'foxy',
```

Comment:
not a big deal for this PR but I wonder if we should eventually separate these into TxParser and "EthSubParser" or something. yearn uniswap, zrx feel like sub parsers of eth to me

Reply:
Yea that's good call, and I agree. But I think that makes sense to do in a separate PR.